backend/server.py

```diff
@@ -43,7 +45,17 @@ async def upload(file: UploadFile = Form(...),
 
         # Delete the temporary chunks directory
         shutil.rmtree(CHUNKS_DIR.parent)
+        
+        # STT Process
```

Comment:
move this to another method, the upload method is long anyway, but the stt is a whole other phase